Package.swift

```diff
@@ -100,3 +100,13 @@ let package = Package(
   cLanguageStandard: .c99,
   cxxLanguageStandard: CXXLanguageStandard.gnucxx14
 )
+
+extension Platform {
+  static var catalyst: Self {
```

Comment:
nice idea!

Reply:
I'm not sure I understand this fix. Is there a particular test that it fixes?

I see this failure building the swift-test target on beta4:

![Screen Shot 2021-07-30 at 1 46 41 PM](https://user-images.githubusercontent.com/73870/127709921-c97f2bc2-79af-4f89-a274-2d774a9e4b0d.png)